src/Integration.Vsix/CFamily/ProcessRunner.cs

```diff
@@ -136,9 +141,23 @@ public bool Execute(ProcessRunnerArguments runnerArgs)
             return succeeded;
         }
 
-        #endregion Public methods
-
-        #region Private methods
+        private static Action KillProcess(Process process)
+        {
+            return () =>
```

Comment:
It looks a bit weird defining `KillProcess` to return an `Action`. 
Is there a reason not to have have `void KillProcess` and call it using `() => KillProcess(...)`?